src/api/app/views/webui/monitor/_events.html.haml

```diff
@@ -0,0 +1,119 @@
+%h2 Statistical Plots
+.grid_7.prefix_1.alpha
+  .box.box-shadow
+    %h2.box-header Options
+    - unless @available_arch_list.empty?
+      = form_tag(controller: "main", action: "index") do
+        %p
+          %strong Architecture:
+          = select_tag(:architecture_display, options_for_select(@available_arch_list, @default_architecture))
+          %br/
+          %strong Timeframe:
+          = select_tag(:time_display, options_for_select([["1 day", "24"], ["1 hour", 1], ["1 week", "168"], ["1 month", "720"], ["1 year", "8760"]], "24"))
+.grid_8.omega
+  #building{ style: "height:250px; margin-right: 20px" }
+  .centered
+    Workers
+    %span#legend_building.flot_legend
+  .plotspinner= image_tag('ajax-loader.gif')
+.clear
+.grid_8.alpha
+  #events{ style: "height:250px; margin-left: 20px; margin-right: 20px" }
+  .centered
+    Repositories to recalculate
+    %span#legend_events.flot_legend
+  .plotspinner= image_tag('ajax-loader.gif')
+.grid_8.omega
+  #jobs{ style: "height:250px; margin-right: 20px" }
+  .centered
+    Packages scheduled for building
+    %span#legend_jobs.flot_legend
+  .plotspinner= image_tag('ajax-loader.gif')
+.clear
+
+= content_for :head_javascript do
+  :plain
```

Comment:
You can read the manual...  😸 
But, is for having plain text (keeping the indentation) inside the HAML, so we are using the `conten_for` helper for including javascript into a tag lately, so the best is using a :plain to include it there, if I use :javascript then I will end up having nested `<script>` tags